accounts-db/src/ancient_append_vecs.rs

```diff
@@ -328,11 +357,20 @@ impl AccountsDb {
         sorted_slots: Vec<Slot>,
         can_randomly_shrink: bool,
     ) {
+        /*
+        // maybe we want this to be a different tuning parameter passed on cli?
+        let max_ancient_slots = if let Some(offset) = self.ancient_append_vec_offset {
+            offset.unsigned_abs().min(10_000) as usize
+        } else {
+            return;
+        };
+        */
+        let max_ancient_slots = 100_000; // this gives us approx 5M ideal size for testing against mnb now
```

Comment:
@HaoranYi I think once other changes are made, this number probably needs to be a cli arg and it should probably default to 100k or so. The question is 432k + this number - ancient offset becomes the expected # of storages we may have open at one time. So, that affects the global mmap or file handle limit. The more files we put here, the smaller size per file. Smaller files are great. Larger files are intense in i/o and cpu to rewrite.

Reply:
yeah. I will add a cli arg for this one.